gui/src/ais.cpp

```diff
@@ -368,6 +386,8 @@ void AISDrawAreaNotices(ocpnDC &dc, ViewPort &vp, ChartCanvas *cp) {
                   GetCanvasPointPix(vp, cp, lat, lon, &target_point);
                   points.push_back(target_point);
                 }
+                dc.DrawLines(points.size(), &points.front());
+                break;
               }
```

Comment:
What is the point of drawing the polygon here? AFAICT it is drawn at the end in https://github.com/OpenCPN/OpenCPN/blob/master/gui/src/ais.cpp#L374

Reply:
This is a polyline. Hence DrawLines.